include/seqan3/alphabet/composition/union_composition.hpp

```diff
@@ -118,6 +119,62 @@ template <typename ...alphabet_types>
 //!\endcond
 class union_composition
 {
+private:
+    //!\brief A meta::list The types of each composite in the composition
+    using composites = meta::list<alphabet_types...>;
+//static_assert(constructible_concept<float, int>);
+    /*!\brief 'Callable' helper class that is invokable by meta::invoke.
+      * Returns an std::true_type if the `type` is constructable from `T`.
+      */
+    template <typename T>
+    struct constructible_from
+    {
+        //!\brief The returned type when invoked.
+        template <typename type>
+        using invoke = std::conditional<std::is_constructible_v<type, T>, std::true_type, std::false_type>;
+        //std::integral_constant<bool, constructible_concept<type, T>>;
```

Comment:
can you try these again? and also double check that the concept does not work?

Reply:
The concept does not work in either syntax while `std::is_constructible_v` does.